acid/features/history/tests/conftest.py

```diff
@@ -11,16 +11,14 @@
 TIME_END = '2018-02-23 23:55:00'
 
 
-def date_to_epoch(t_stamp):
+def datetime_to_seconds(t_stamp):
     return int(t_stamp.strftime("%s")) if t_stamp else None
 
-
-def epoch_to_date(t_epoch):
+def seconds_to_datetime(t_epoch):
```

Comment:
Similar functions are in https://github.com/codilime/acid/blob/master/acid/features/status/time_utils.py
Maybe we want to move time_utils to ACID scope and expose them to all features?